src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+public class Message{
+    public static String sender;
+    public static String message_body;
+    public static Long timestamp;
```

Comment:
don't use static, static is used for single values that are not unique to the object itself.

AKA, there is only one copy of sender, messageBody, and timestamp